packages/builder/src/pages/builder/portal/manage/users/_components/AddUserModal.svelte

```diff
@@ -49,17 +74,27 @@
   <Select
     placeholder={null}
     bind:value={selected}
-    on:change
     {options}
     label="Add new user via:"
   />
-  <Input
-    type="email"
-    bind:value={$email}
-    error={$touched && $error}
-    placeholder="john@doe.com"
-    label="Email"
-  />
+
+  {#if basic}
+    <Input
+      type="email"
+      label="Email"
+      bind:value={$email}
+      error={$touched && $error}
+    />
+    <Input disabled label="Password" value={password} />
+  {:else}
+    <Input
+      type="email"
+      bind:value={$email}
+      error={$touched && $error}
+      placeholder="john@doe.com"
+      label="Email"
+    />
+  {/if}
```

Comment:
The email inputs are identical here aren't they? So we could just define one of them, and move the `{#if basic}` to after that, and then just render the password input if they want basic onboading? Just to reduce duplication.